content/bdd-with-cucumber/08/08.03/index.adoc

```diff
@@ -66,4 +66,4 @@ We don't write Gherkin during an example mapping session, so that's not one of t
 
 **Explanation:**
 
-In project management, there are famously "uknown unknowns", "known unknowns" and "known knowns". The most dangerous are the "unknown unknows" because not only do we not know the answer to them, we have not even realised yet that there's a question!
\ No newline at end of file
+In project management, there are famously "uknown unknowns", "known unknowns" and "known knowns". The most dangerous are the "unknown unknows" because not only do we not know the answer to them, we have not even realised yet that there's a question!
```

Comment:
```suggestion
In project management, there are famously "unknown unknowns", "known unknowns" and "known knowns". The most dangerous are the "unknown unknows" because not only do we not know the answer to them, we have not even realised yet that there's a question!
```